rules/no-object-as-default-parameter.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+
+const MESSAGE_ID = 'noObjectAsDefaultParameter';
+
+const objectParameterSelector = [
+	':function > AssignmentPattern.params',
+	'[left.type="Identifier"]',
+	'[right.type="ObjectExpression"]',
+	'[right.properties.length>0]'
+].join('');
+
+const create = context => {
+	return {
+		[objectParameterSelector]: node => {
+			context.report({
+				node,
```

Comment:
@medusalix  Do you think we should report on parameter name? `node.left`

Reply:
Yes, makes sense.